src/components/Breadcrumbs/Breadcrumbs.stories.tsx

```diff
@@ -132,6 +132,15 @@ export const ColorVariations: Story = {
           breadcrumbsType="whatsOn"
         />
       </Box>
+      <Box>
+        <Heading level="h3" size="heading6">
+          Digital Collections
+        </Heading>
+        <Breadcrumbs
+          breadcrumbsData={breadcrumbsData}
+          breadcrumbsType="digitalCollections"
+        />
+      </Box>
```

Comment:
To keep the full list alphabetized. move this example between `Connect` and `Education`.